clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -83,10 +86,14 @@
    * @param options the client options
    */
   public FileInStream(FileInfo info, InStreamOptions options) {
+    this(info, options, FileSystemContext.INSTANCE);
+  }
+
+  FileInStream(FileInfo info, InStreamOptions options, FileSystemContext context) {
```

Comment:
Any tests can use PowerMock to modify final members of classes. Take a look at CreateOptionsTests for an example.

You should not need to create a separate constructor for the sake of passing in the context.